### DISCUSSION TITLE
Symbolアートや人気NFTの情報求む

### DISCUSSION THREAD

**Discussion by ymuichiro** [ymuichiro/symbol_japan_forum#20] [Ideas]
以下の情報をご提供いただけないでしょうか

- アートのURL
- 作者のTwitterアカウント
- 本Github Wiki への貼り付け可否

**Comment by ghost** [comment]
・アートのURL
http://nft-drive-data-explorer.tk/data.html?id=326BABB2CF5EB1ED&w=90vh

・作者のTwitterアカウント
[@honanem](https://twitter.com/honanem)

・本Github Wiki への貼り付け可否
OK

**Reply by ymuichiro** [reply]
勝手に名前つけちゃったけど、画像のタイトルは ねこ でよかったかな
https://github.com/ymuichiro/symbol_japan_forum/wiki/Gallery

**Comment by 0x070696E65** [comment]
- アートのURL https://comsa.io/nft/02b5bbf3-17a7-4084-b33a-a643b79a1350
- [@komiya_ma](https://twitter.com/komiya_ma)
- 本Github Wiki への貼り付け可否 : OK

**Comment by 0x070696E65** [comment]
- アートのURL https://comsa.io/nft/c95a7586-46b6-4de7-b0cd-00ebf1e5e3a7
- [@komiya_ma](https://twitter.com/komiya_ma)
- 本Github Wiki への貼り付け可否 : OK

**Comment by kanekodaisuk** [comment]
・アートのURL
https://comsa.io/nft/0787e346-65b9-404a-9d30-3c0a7b9b9d0f
・@Kaneko_daisuke
・本Github wikiへの貼り付け可否:OK

**Comment by ymuichiro** [comment]
・アートのURL
https://explorer.comsa.io/mosaic/1C729477CC488B68
・@faunsu19000
・本Github wikiへの貼り付け可否:OK
